Rakefile

```diff
@@ -125,6 +127,16 @@ task :student do
   Rake::Task["post"].execute
 end
 
+## This requires some configs 
+desc "Package and ship a VM"
+task :ship do
+  vmname = ENV['VMNAME'].split(".").first
+  ovapath=retrieve_ova(vmname)
+  push_vmware(ovapath)
+  ship_vm_to_dir(ovapath, "/mnt/nfs/EducationVMS/#{ENV['VMTYPE']}")
```

Comment:
Right now it's tied to pe-vm-builder-1, but yeah it would have to be tied to a set of builders that had that nfs export mounted.

We can add other ship methods later (rsync, s3, whatevs) ass needed